src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random(System.currentTimeMillis());
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int randomMaxValue = 100;
+    private final int defaultRadius = 10;

```

Comment:
Please follow the rules of constants naming
```suggestion
    private static final int RANDOM_MAX_VALUE = 100;
    private static final int DEFAULT_RADIUS = 10;
```